src/pages/fr/core-concepts/sharing-state.md

```diff
@@ -0,0 +1,698 @@
+
+```js
+// sharedStore.js
+import { createSignal } from 'solid-js';
+
+export const sharedCount = createSignal(0);
+```
+...et tous les composants important `sharedCount` partageront le même état. Bien que cela fonctionne bien, vous préférerez peut-être Nano Stores si a) vous aimez leurs modules complémentaires pour
+ ["objects"](https://github.com/nanostores/nanostores#maps) et [async state](https://github.com/nanostores/nanostores#lazy-stores), ou b) vous souhaitez communiquer entre Solid et d'autres frameworks d'interface utilisateur comme Preact ou Vue.
+
+</details>
+:::
+
+## Installer Nano Stores
+
+Pour commencer, installez Nano Stores avec leur package d'assistance pour votre framework d'interface utilisateur préféré:
+
+<UIFrameworkTabs>
+  <Fragment slot="preact">
+  ```shell
+  npm install nanostores @nanostores/preact
+  ```
+  </Fragment>
+  <Fragment slot="react">
+  ```shell
+  npm install nanostores @nanostores/react
+  ```
+  </Fragment>
+  <Fragment slot="solid">
+  ```shell
+  npm install nanostores @nanostores/solid
+  ```
+  </Fragment>
+  <Fragment slot="svelte">
+  ```shell
+  npm install nanostores
+  ```
+  :::note
+  Pas de package d'assistance ici ! Nano Stores peuvent être utilisés comme Svelte stores standard
+
+  :::
+  </Fragment>
+  <Fragment slot="vue">
+  ```shell
+  npm install nanostores @nanostores/vue
+  ```
+  </Fragment>
+</UIFrameworkTabs>
+
+Vous pouvez aller directement dans le [Guide d'utilisation de Nano Stores](https://github.com/nanostores/nanostores#guide) à partir d'ici, ou suivez notre exemple ci-dessous!
+
+## Exemple d'utilisation - Menu déroulant pour un panier de ecommerce
+
+Disons que nous construisons une interface de ecommerce simple avec trois éléments interactifs :
+-Un formulaire de soumission "ajouter au panier"
+-Un menu déroulant du panier pour afficher les articles ajoutés
+-Une bascule de chariot
+
+<LoopingVideo sources={[{ src: '/public/videos/stores-example.mp4', type: 'video/mp4' }]} />
+
+_[**Essayez l'exemple complété**](https://github.com/withastro/astro/tree/main/examples/with-nanostores) sur votre machine ou en ligne via Stackblitz._
+
+Votre fichier Astro de base peut ressembler à ceci:
+
+```astro
+---
+// Exemple: src/pages/index.astro
+import CartFlyoutToggle from '../components/CartFlyoutToggle';
+import CartFlyout from '../components/CartFlyout';
+import AddToCartForm from '../components/AddToCartForm';
+---
+
+<!DOCTYPE html>
+<html lang="en">
+<head>...</head>
+<body>
+  <header>
+    <nav>
+      <a href="/">Astro storefront</a>
+      <CartFlyoutToggle client:load />
+    </nav>
+  </header>
+  <main>
+    <AddToCartForm client:load>
+    <!-- ... -->
+    </AddToCartForm>
+  </main>
+  <CartFlyout client:load />
+</body>
+</html>
+```
+
+### Utilisant "atoms"
+
+Commençons par ouvrir notre `Cart Flyout` chaque fois que `Car Flyout Toggle` est cliqué. 
```

Comment:
We don't need spaces here
```suggestion
Commençons par ouvrir notre `CartFlyout` chaque fois que `CarFlyoutToggle` est cliqué. 
```

Reply:
Adding that there’s a missing `t` in the second one:

```suggestion
Commençons par ouvrir notre `CartFlyout` chaque fois que `CartFlyoutToggle` est cliqué. 
```